pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -3544,6 +3544,31 @@ The paths themselves are given in the direction of the first geometry.`,
 			Volatility: tree.VolatilityImmutable,
 		},
 	),
+	"st_rotate": makeBuiltin(
+		defProps(),
+		tree.Overload{
+			Types: tree.ArgTypes{
+				{"g", types.Geometry},
+				{"rotRadians", types.Float},
+			},
+			ReturnType: tree.FixedReturnType(types.Geometry),
+			Fn: func(_ *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+				g := args[0].(*tree.DGeometry)
```

Comment:
I made a recent commit that fixed most of the previous comments, except for this one. Since I was missing the tree.MustBeDGeometry/tree.MustBeDFloat types in my forked repo at the time.

Reply:
they should definitely be there.

Reply:
fixed in recent commit, I also rebased my fork onto the upstream and squashed my past commits into 1.